### PR TITLE
[1LP][RFR] Test coverage for BZ 1683697

### PR DESCRIPTION
**Pull request by john-dupuy**
Adding a new automated test for [BZ 1683697](https://bugzilla.redhat.com/show_bug.cgi?id=1683697). 

{{ pytest: --use-template-cache cfme/tests/control/test_bugs.py::test_accordion_after_condition_creation }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit d797091263337fd58f5a8c9a0b19f602192f05af:

`cfme/tests/control/test_bugs.py`:
- :bangbang: Line 455:27: **E127** *continuation line over-indented for visual indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/control/test_bugs.py [hunk 1]
```diff
@@ -440,3 +449,48 @@ def _check_query():
...
+        ))
+    condition = condition_class(
+        appliance.collections.conditions,
+        fauxfactory.gen_alpha(),
+        expression="fill_field({} : Name, IS NOT EMPTY)".format(
+            condition_class.FIELD_VALUE)
+    )
+    view = navigate_to(condition, "Add", force=True)
+    # first fill description, expression, and notes
+    view.fill({
```
Comment (sshveta) [inline cfme/tests/control/test_bugs.py]: Required : Please use create method instead

Comment (john-dupuy) [reply cfme/tests/control/test_bugs.py]: @sshveta most recent push has been updated to use the create method.

File: cfme/tests/control/test_bugs.py [hunk 2]
```diff
@@ -7,12 +7,13 @@
...
 
 from cfme import test_requirements
+from cfme.control.explorer import conditions
 from cfme.control.explorer import ControlExplorerView
 from cfme.control.explorer.alert_profiles import ServerAlertProfile
-from cfme.control.explorer.conditions import VMCondition
 from cfme.control.explorer.policies import VMCompliancePolicy
 from cfme.control.explorer.policies import VMControlPolicy
 from cfme.exceptions import CFMEExceptionOccured
+from cfme.tests.control.test_basic import CONDITIONS
```
Comment (izapolsk) [inline cfme/tests/control/test_bugs.py]: Could you move such constants used in many test suits to some common place ?
cross references between test suits are frowned upon.
I belive you can move them to
```
cfme/tests/control/__init__.py
or 
cfme/control/__init__py
```

Comment (izapolsk) [reply cfme/tests/control/test_bugs.py]: ignore above

**Pull request merged**